windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
Ok. I used ubuntu instead of node-slim here to ease installation of the dependency imaging libraries. Hopefully they're interchangeable then.